metricbeat/module/logstash/logstash_integration_test.go

```diff
@@ -85,6 +85,8 @@ func TestXPackEnabled(t *testing.T) {
 
 		event := events[0]
 		require.Equal(t, metricSetToTypeMap[metricSet.Name()], event.RootFields["type"])
+		//require.Equal(t, "TODO", event.RootFields["cluster_uuid"])
+		t.Logf("***** cluster_uuid: ", event.RootFields["cluster_uuid"])
```

Comment:
I will be cleaning this up in this PR. Just want to see how much work it might be to automate the testing of this PR.